images/lang/en.yml

```diff
@@ -3,9 +3,9 @@ en:
     MENUTITLE: SilverStripe\SessionManager\Controllers\LoginSessionController
     REMOVE_FAILURE: 'Something went wrong.'
     REMOVE_PERMISSION: 'You do not have permission to delete this record.'
-    REMOVE_SUCCESS: 'Successfully logged out of device.'
+    REMOVE_SUCCESS: 'Successfully removed session.'
```

Comment:
There shouldn't be a `lang` folder under images.

Reply:
No idea why, but I've changed it.